docs/gettingStarted/customScheme/clientForNewScheme.md

```diff
@@ -0,0 +1,378 @@
+           if (options.where[key] === undefined) {
+             continue
+           }
+
+           if (key === 'member' || key === 'dao') {
+             const option = options.where[key] as string
+             isAddress(option)
+             options.where[key] = option.toLowerCase()
+           } else {
+             where += `${key}: "${options.where[key] as string}"\n`
+           }
+         }
+
+         const query = gql`query DepositSearch
+           {
+             deposits ${createGraphQlQuery(options, where)} {
+               id
+               member
+               amount
+               avatar
+               rep
+             }
+           }
+         `
+
+         return context.getObservableListWithFilter(
+           query,
+           (r: any) => {
+             return new Deposit({
+               id: r.id,
+               member: r.member,
+               amount: new BN(r.amount || 0),
+               dao: r.avatar,
+               rep: new BN(r.rep || 0)
+             }, context)
+           },
+           daoFilter,
+           apolloQueryOptions
+         ) as Observable<Deposit[]>
+       }
+
+       public id: string|undefined
+       public staticState: IDepositStaticState|undefined
+
+       constructor(idOrOpts: string|IDepositStaticState, public context: Arc) {
+         if (typeof idOrOpts === 'string') {
+           this.id = idOrOpts
+         } else {
+           const opts = idOrOpts as IDepositStaticState
+           this.id = opts.id
+           this.setStaticState(opts)
+         }
+       }
+
+       public setStaticState(opts: IDepositStaticState) {
+         this.staticState = opts
+       }
+
+       public async fetchStaticState(): Promise<IDepositStaticState> {
+         if (!!this.staticState) {
+           return this.staticState
+         } else {
+           return await this.state().pipe(first()).toPromise()
+         }
+       }
+
+       public state(apolloQueryOptions: IApolloQueryOptions = {}): Observable<IDepositState> {
+         const query = gql`query DepositById {
+           deposit (id: "${this.id}") {
+             id
+             memberAddress
+             amount
+             avatar
+             rep
+           }
+         }`
+
+         const itemMap = (item: any): IDepositState => {
+           if (item === null) {
+             throw Error(`Could not find a Vote with id ${this.id}`)
+           }
+           return {
+             amount: item.amount,
+             dao: item.dao,
+             id: item.id,
+             member: item.member,
+             rep: item.rep
+           }
+         }
+         return this.context.getObservableObject(query, itemMap, apolloQueryOptions)
+       }
+    }
+
+## Integration Tests
+  1. Add relevant integration test for the new scheme, `client/test/scheme-buyInWithRageQuitOpt.spec.ts`
+  2. Start test watcher while you test and update the client
+
+        npm run test:watch:client -- test/scheme-buyInWithRageQuitOpt.spec.ts
```

Comment:
there is a funny character being displayed here

Reply:
What is that funny character??
If it is a backwards `p` it is for the link 
If it is `two squares` that is for copy the command
else I'm not sure

Reply:
You can see it by clicking "View File" in the rendered markdown:

![image](https://user-images.githubusercontent.com/1821666/66919664-c0a87b00-efef-11e9-9930-51a5d78c9aa9.png)